math/physics/src/TLorentzVector.cxx

```diff
@@ -5,9 +5,22 @@
 //    Oct 20 1999: dito in Double_t operator()
 //    Jan 25 2000: implemented as (fP,fE) instead of (fX,fY,fZ,fE)
 
-
 /** \class TLorentzVector
     \ingroup Physics
+
+## Disclaimer
+In order to represent 4-vectors, TLorentzVector shall not be used.
```

Comment:
"Shall not be used" is too strong a wording, see e.g. https://english.stackexchange.com/questions/36228/definition-and-meanings-of-shall-not

We could use "should". But maybe instead "`TLorentzVector` is a legacy class; it is slower and worse for serialization than the recommended superior `ROOT::Math::LorentzVector` classes."